manifests/m/maforhet/ComicRackCE/nightly/maforhet.ComicRackCE.installer.yaml

```
@@ -0,0 +1,26 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: maforhet.ComicRackCE
+PackageVersion: nightly

```

```suggestion
PackageVersion: nightly-9745855
```
This should match what is shown in Control Panel.
After updating this field, the version folder will need to be renamed